spec/spec_helper.rb

```diff
@@ -1,16 +1,45 @@
+
 require 'puppetlabs_spec_helper/module_spec_helper'
-require 'simplecov'
```

Comment:
The change in this file will remove your code coverage, just thought I'd call this out incase it was unintended.

Reply:
@HelenCampbell Thanks for the valuable comments. Incorporated the review comment.